types/deploy.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+type Deploy struct {
+	IdModel
+	AppId   int64  `json:"appId"`
```

Comment:
Make sure to add `StackId` in here.

Reply:
Done, I also added the OrgName since enigma includes it on the model as well.